scripts/library/DOM.js

```diff
@@ -0,0 +1,28 @@
+//@ts-check
+export default class DOMHandler {
+  /**
+   * html: <p>태그 추가(after)
+   * @param {string} query - cssSelector
+   * @param {string} className - 생성된 태그 class명
+   * @param {string} text - 추가할 text
+   */
+  static addTextAfter = (query, className, text) => {
+    const element = document.querySelector(query);
+    if (element) {
+      const textTag = document.createElement('p');
+      textTag.textContent = text;
+      textTag.className = className.slice(1);
+      element.after(textTag);
+    }
```

Comment:
이렇게 if문으로 끝나는 구문에서는 [early return](https://devscb.tistory.com/130)을 사용해보시면 좋습니다.

```suggestion
    const element = document.querySelector(query);
    if(!element) return;
     const textTag = document.createElement('p');
     textTag.textContent = text;
     textTag.className = className.slice(1);
     element.after(textTag);
    }
```
얼리 리턴은 유명한 리팩토링 기법인데요, 
if문을 사용하여 indent(코드 들여쓰기)가 깊어지면 가독성 측면에서 좋지 않기도 하고, 미세하게 코드 성능도 안좋기도 합니다. 그래서 이럴 때 빠른 return 문을 사용해서 코드를 개선해 볼 수 있어요.

if문을 사용하는 다른 곳도 마찬가지입니다!

Reply:
헙! 처음에 이렇게 구현했었는데 뭔가 낫을 붙이는게 안좋은건가 싶어서 정정했었는데! 이번 기회에 확실히 알게 되었습니다! 정정했습니다!
https://github.com/codeit-bootcamp-frontend/4-Weekly-Mission/pull/175/commits/776ead387b0077555578d38d90a6da1dad954131

Reply:
준영님 말씀처럼 부정연산자를 많이 쓰는건 좋지 않은게 맞습니다. 👍 잘알고계시네요!
다만 얼리리턴할때는 예외로 부정연산자를 일반적으로 잘 사용하는 것 같아요ㅎㅎ비교해봤을 때 어떤게 가독성이 더 좋은지 한번 보시고, 준영님만의 기준을 만들어보셔도 괜찮을 것 같아요 :)